### PR TITLE
Merge 2.0.00 into develop

### PR DESCRIPTION
**Pull request by mark-prins**
# 👩🏻‍💻 What does this PR do?

Merge `main` back into `develop` - now that 2.0.00 has been merged to main and the release branch closed, time to get the last minute changes back into dev.

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] Run the unit tests
- [ ] Run the f/e tests

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [x] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx [hunk 1]
```diff
@@ -171,12 +171,11 @@ export const NumericTextInput: FC<NumericTextInputProps> = React.forwardRef(
           onChange(newNum);
         }}
         onBlur={() => {
-          if (isDirty) {
```
Comment (lache-melvin) [inline client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx:174]: We shouldn't lose this `isDirty` check

**Review by roxy-dao — APPROVED** [review]
LGTM~

**Review by lache-melvin — CHANGES_REQUESTED** [review]
One little fix for the NumericTextInput 🙏  Not super familiar with the asset changes, if there was anything finnicky there, but everything else looks good to me!

File: client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx (hunk 1 above)
Comment (mark-prins) [reply client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx:174]: good catch! thanks - there was a conflict in this component, though not at this point. this one is a plain GH merge thing..

**Comment by jmbrunskill** [comment]
> One little fix for the NumericTextInput 🙏 Not super familiar with the asset changes, if there was anything finnicky there, but everything else looks good to me!

Asset stuff looks ok to me, I'm not sure when the `warnings` thing went in but I assume it's come through correctly...

File: client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx (hunk 1 above)
Comment (mark-prins) [reply client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx:174]: Ah yeah. Two PRs (#3884 & #3825) both targeting the same area in the same file, one in develop and the other in the release branch. Dangerous ground!

image

**Review by lache-melvin — APPROVED** [review]
Thank you!

**Comment by mark-prins** [comment]
Thanks everyone!! ❤️

**Pull request merged**